kyo-bench/src/main/scala/kyo/bench/ForkJoinContentionBench.scala

```diff
@@ -0,0 +1,57 @@
+package kyo.bench
+
+import org.openjdk.jmh.annotations.*
+
+class ForkJoinContentionBench extends Bench.ForkOnly[Unit]:
+
+    val depth     = 1000
+    val parallism = Runtime.getRuntime().availableProcessors()
```

Comment:
Do you know what value this evaluated to on your machine? Do we need a minimum number here?

Reply:
It returns `12` on my M2 Max. Setting to the number of cores helps to keep the benchmark testing similar behavior regardless of how many cores the machine has.